.github/workflows/issues.yaml

```
@@ -16,10 +16,6 @@ jobs:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
           configuration-path: .github/triage-labeler.yaml
           enable-versioned-regex: 0
-
-  add-to-project:
-    runs-on: ubuntu-latest
-    steps:
       - uses: actions/add-to-project@v0.4.1
         with:
           github-token: "${{ secrets.ADD_TO_PROJECT_PAT }}"
```

:question: should this get updated to use meowblecoin's pat?
```suggestion
          github-token: "${{ secrets.MEOWBLECOIN_PAT }}"
```